src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -50,7 +50,7 @@ public void startExecuting()
         BlockPos pos = citizen.getHomePosition();
         if (pos == null)
         {
-            Log.getLogger().error("EntityCitizen has null townHall (And no home)");
+            //todo: do something about this instead of spamming console
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)